google/cloud/bigquery/table.py

```diff
@@ -1799,6 +1806,7 @@ def to_dataframe(
         dtypes=None,
         progress_bar_type=None,
         create_bqstorage_client=True,
+        date_as_object=True,
```

Comment:
Ah, sorry for initially missing that - let's add a docstring for the new parameter here, too.

Reply:
I think it's there https://github.com/googleapis/python-bigquery/pull/150/files/7abb6e0ec64c55023df6fbb0809abe1f8cf6ec3e#diff-f0febbf80214497b41d3dba7fe3b8d37R1687-R1692

Reply:
That's a different one. :)

This particular `to_dataframe()` method is in the `_EmptyRowIterator` class and the new parameter does not seem to be documented a few liens below.